src/coreclr/src/jit/block.h

```diff
@@ -448,6 +448,7 @@ struct BasicBlock : private LIR::Range
 
 #define BBF_BACKWARD_JUMP_TARGET           MAKE_BBFLAG(36) // Block is a target of a backward jump
 #define BBF_PATCHPOINT                     MAKE_BBFLAG(37) // Block is a patchpoint
+#define BBF_HAS_VIRTUAL_CALL               MAKE_BBFLAG(38) // BB contains a call needing a class profile
```

Comment:
Aren't you doing profiling on both virtual and interface calls? Should this be named `BBF_NEEDS_CLASS_PROFILE`?